app/views/galleries/_single.haml

```diff
@@ -47,7 +47,7 @@
                 %br
                 %span.icon-keyword= keyword
               - if is_owner && !skip_forms
-                .select-button= check_box_tag :"marked_ids[]", icon_id
+                .select-button= check_box_tag :"marked_ids[]", icon_id, true, id: nil
```

Comment:
This should default to false – currently defaults to making all icon check boxes ticked on gallery#show